invenio_theme/assets/semantic-ui/less/invenio_theme/theme/navbar/navbar.less

```diff
@@ -0,0 +1,233 @@
+nav {
```

Comment:
question: all files in the theme folder are suffixed with `.overrides` extension. But this is a mere `.less`. Being this the case, shouldn't it go to the `theme.less` file? I am not deeply familiar with theming in semantic, so I just asking.

Reply:
Hmm, since there is no other `.less` file in this module and all the rules apply for the navbar I thought it should be named as `navbar.less` and imported in the `site.overrides`. If you are asking if we should import it into the `theme.less` at the end this file is being imported in the `site.overrides` ( you can check this in cookiecutter-invenio-instance, see [here](https://github.com/inveniosoftware/cookiecutter-invenio-instance/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/%7B%7Bcookiecutter.package_name%7D%7D/theme/assets/semantic-ui/less/%7B%7Bcookiecutter.package_name%7D%7D/site/globals/site.overrides) )